### PR TITLE
SAM-2751 Samigo honor pledge setting not being persisted

### PR DESCRIPTION
**Pull request by ern**

### CODE REVIEW

**Comment by juanjmerono** [comment]
Please, solve conflicts here, probably due to this merge:
https://github.com/sakaiproject/sakai/pull/2350

File: reference/docs/conversion/sakai_11_oracle_conversion.sql [hunk 1]
```diff
@@ -969,3 +969,12 @@ INSERT INTO SAM_ASSESSMETADATA_T (ASSESSMENTMETADATAID, ASSESSMENTID, LABEL, ENT
       AND TYPEID='142' AND ISTEMPLATE=1),
        'instructorNotification_isInstructorEditable', 'true');
 --END SAM-2709
+
+-- SAM-2751
+ALTER TABLE SAM_ASSESSACCESSCONTROL_T ADD HONORPLEDGE BIT;
```
Comment (juanjmerono) [inline reference/docs/conversion/sakai_11_oracle_conversion.sql]: BIT is not a valid type in Oracle, change this sentence with:
ALTER TABLE SAM_ASSESSACCESSCONTROL_T ADD HONORPLEDGE NUMBER(1,0);

File: reference/docs/conversion/sakai_11_oracle_conversion.sql [hunk 2]
```diff
@@ -969,3 +969,12 @@ INSERT INTO SAM_ASSESSMETADATA_T (ASSESSMENTMETADATAID, ASSESSMENTID, LABEL, ENT
...
+ALTER TABLE SAM_PUBLISHEDACCESSCONTROL_T ADD HONORPLEDGE BIT;
```
Comment (juanjmerono) [inline reference/docs/conversion/sakai_11_oracle_conversion.sql]: BIT is not a valid type in Oracle, change this sentence with:
ALTER TABLE SAM_PUBLISHEDACCESSCONTROL_T ADD HONORPLEDGE NUMBER(1,0);

**Comment by ern** [comment]
Thanks Juanjo I used an online SQL mysql to oracle converter guess it didn't do very well!

**Pull request merged**